public/data/AntonyKimani.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ANTONY KIMANI",
+    "type": "personal",
+    "bio": "Frontend Web Developer | Open Source Enthusiast | Blogger ",
+    "avatar": "https://github.com/AntonyKimani.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/AntonyKimani",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kimani_dev",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Web Development",
+        "date": "September 2021",
+        "color": "grey",
+        "description": "Discovered Web Development and currently learning JavaScript"
+      }
+    ],
+
+    "milestones": [
+    {
+        "title": "Open Source Contributer",
+        "date": "January 2022",
+        "color": "grey",
+        "description": "Learning Git and Github , actively contributing to Open Source and Collaborations"
+    }
+    ]
+  }
```

Comment:
```suggestion
  "milestones": [
    {
      "title": "Web Development",
      "date": "September 2021",
      "color": "grey",
      "description": "Discovered Web Development and currently learning JavaScript"
    },
    {
      "title": "Open Source Contributer",
      "date": "January 2022",
      "color": "grey",
      "description": "Learning Git and Github , actively contributing to Open Source and Collaborations"
    }
  ]
}

```

Reply:
Hello Krishguptadev , I've seen your changes . What should I do now?